### DISCUSSION TITLE
[Post v1] React 17 compat

### DISCUSSION THREAD

**Discussion by ljosberinn** [chakra-ui/chakra-ui#1869] [General] [answered]
https://github.com/babel/babel/pull/12017#issuecomment-682752525 react 17 will come in a few weeks, we should prepare for chakra-ui being compatible after launching v1.

Last time I checked, `react-focus-lock` has issues with it

**Comment by chasinhues** [comment, marked as answer]
Resolving this discussion as Chakra is now compatible with React 17 👍
